.github/workflows/ogf_workflow.yml

```diff
@@ -102,9 +102,9 @@ jobs:
           ./k6 version
 
       - name: Setup Azure RG and Storage Account For Functions App
-        uses: Azure/cli@1.0.4
+        uses: azure/CLI@v1
         with:
-          azcliversion: 2.16.0
+          azcliversion: latest
```

Comment:
should we lock to specific version? in case some buggy latest is released our pipeline can break .. we can always manually update the version